nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -71,7 +71,7 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [nimdow](https://github.com/avahe-kellenberger/nimdow), a window manager written in Nim, inspired by dwm.
 
-- [woodpecker-agent](https://woodpecker-ci.org/), a simple CI engine with great extensibility. Available as [services.woodpecker-agent](#opt-services.woodpecker-agent.enable).
+- [woodpecker-agents](https://woodpecker-ci.org/), a simple CI engine with great extensibility. Available as [services.woodpecker-agent](#opt-services.woodpecker-agents.enable).
```

Comment:
Don't we need to add a name between woodpecker-agents and enable?

Reply:
`enable` is not per-agent (not in `agentModule` but directly in `services.woodpecker-agents`).

Though I do notice a typo

```suggestion
- [woodpecker-agents](https://woodpecker-ci.org/), a simple CI engine with great extensibility. Available as [services.woodpecker-agents](#opt-services.woodpecker-agents.enable).
```